src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -0,0 +1,218 @@
+     * @Method({"GET", "POST"})
+     *
+     * NOTE: the Method annotation is optional, but it's a recommended practice
+     * to constraint the HTTP methods each controller responds to (by default
+     * it responds to all methods).
+     */
+    public function newAction(Request $request)
+    {
+        $post = new Post();
+        $post->setAuthorEmail($this->getUser()->getEmail());
+
+        // See http://symfony.com/doc/current/book/forms.html#submitting-forms-with-multiple-buttons
+        $form = $this->createForm('AppBundle\Form\PostType', $post)
+            ->add('saveAndCreateNew', 'Symfony\Component\Form\Extension\Core\Type\SubmitType');
+
+        $form->handleRequest($request);
+
+        // the isSubmitted() method is completely optional because the other
+        // isValid() method already checks whether the form is submitted.
+        // However, we explicitly add it to improve code readability.
+        // See http://symfony.com/doc/current/best_practices/forms.html#handling-form-submits
+        if ($form->isSubmitted() && $form->isValid()) {
+            $post->setSlug($this->get('slugger')->slugify($post->getTitle()));
+
+            $entityManager = $this->getDoctrine()->getManager();
+            $entityManager->persist($post);
+            $entityManager->flush();
+
+            // Flash messages are used to notify the user about the result of the
+            // actions. They are deleted automatically from the session as soon
+            // as they are accessed.
+            // See http://symfony.com/doc/current/book/controller.html#flash-messages
+            $this->addFlash('success', 'post.created_successfully');
+
+            if ($form->get('saveAndCreateNew')->isClicked()) {
+                return $this->redirectToRoute('admin_post_new');
+            }
+
+            return $this->redirectToRoute('admin_post_index');
+        }
+
+        return $this->render('admin/blog/new.html.twig', array(
+            'post' => $post,
+            'form' => $form->createView(),
+        ));
+    }
+
+    /**
+     * Finds and displays a Post entity.
+     *
+     * @Route("/{id}", requirements={"id" = "\d+"}, name="admin_post_show")
+     * @Method("GET")
+     */
+    public function showAction(Post $post)
+    {
+        // This security check can also be performed:
+        //   1. Using an annotation: @Security("post.isAuthor(user)")
+        //   2. Using a "voter" (see http://symfony.com/doc/current/cookbook/security/voters_data_permission.html)
+        if (null === $this->getUser() || !$post->isAuthor($this->getUser())) {
+            throw $this->createAccessDeniedException('Posts can only be shown to their authors.');
+        }
+
+        $deleteForm = $this->createDeleteForm($post);
+
+        return $this->render('admin/blog/show.html.twig', array(
+            'post'        => $post,
+            'delete_form' => $deleteForm->createView(),
+        ));
+    }
+
+    /**
```

Comment:
__[development][code_sniffer]__
* Doc comment for parameter "$post" missing
* Doc comment for parameter "$request" missing

<!-- Comment by [SideCI](https://sideci.com) -->